parser/parser.go

```diff
@@ -109,6 +114,9 @@ func ParseFile(r io.Reader) (*File, error) {
 				case "message":
 					// transition to stateMessage which validates the message role
 					next = stateMessage
+					cmd.Name = s
+				case "ollama":
+					next = stateVersion
```

Comment:
I'm not sure a state is entirely necessary. next state can be a normal value and validated when the value is committed

Reply:
Is it better to just have a big if inside `stateValue` then?